python/tests/test_typedef.py

```diff
@@ -46,7 +51,8 @@ def test_record_repr() -> None:
 
 
 def test_named_record() -> None:
-    r = Record(fields=(NestedField(0, "id", IntegerType()), NestedField(1, "name", StringType())))
+    r = Record()
+    r.set_record_schema(StructType(NestedField(0, "id", IntegerType()), NestedField(1, "name", StringType())))
```

Comment:
I think I prefer passing the schema into the constructor. We may want to keep doing that, at least for the generic record.

Reply:
Changed!